qdrant-landing/themes/qdrant-2024/layouts/partials/core-features.html

```diff
@@ -1,27 +1,33 @@
 <section class="core-features">
-  <h2 class="core-features__title">{{ .Params.title }}</h2>
-  <p class="core-features__subtitle">{{ .Params.description }}</p>
-  <a class="link link_xl" href="{{ .Params.link.url }}">{{ .Params.link.text }}</a>
-  <div class="card core-features__cloud-native">
-    <div>
-      <h4 class="core-features__card-title">{{ .Params.cloudFeature.title }}</h4>
-      <p class="core-features__card-content">{{ .Params.cloudFeature.content }}</p>
-      <a class="link link_dark link_md" href="{{ .Params.cloudFeature.link.url }}">
-        {{ .Params.cloudFeature.link.text }}
-      </a>
+  <div class="container">
```

Comment:
Is the default bootstrap `.container` width match our design? If it's not it can be [adjusted](https://getbootstrap.com/docs/5.1/layout/containers/#sass) in `qdrant-landing/themes/qdrant-2024/assets/css/_theme-variables.scss` like this:
``` scss
$container-max-widths: (
  sm: 540px,
  md: 720px,
  lg: 960px,
  xl: 1140px,
  xxl: 1320px
);
```